spec/rubocop/cop/style/braces_around_hash_parameters_spec.rb

```diff
@@ -181,55 +181,55 @@
       )
       RUBY
       corrected = autocorrect_source(src)
-      expect(corrected).to eq(<<-RUBY.strip_indent)
+      expect(corrected).to eq(<<-RUBY)
       foo(
-        qux: "bar",
+          qux: "bar",
           baz: "bar",
           thud: "bar"
       )
       RUBY
     end
 
     it 'corrects brace removal with lower extra line' do
-      src = <<-RUBY.strip_indent
+      src = <<-RUBY
       foo({
         baz: 7
         }
       )
       RUBY
       corrected = autocorrect_source(src)
-      expect(corrected).to eq(<<-RUBY.strip_indent)
+      expect(corrected).to eq(<<-RUBY)
       foo(
         baz: 7
       )
       RUBY
     end
 
     it 'corrects brace removal with top extra line' do
-      src = <<-RUBY.strip_indent
+      src = <<-RUBY
       foo(
         {
           baz: 5
       })
       RUBY
       corrected = autocorrect_source(src)
-      expect(corrected).to eq(<<-RUBY.strip_indent)
+      expect(corrected).to eq(<<-RUBY)
       foo(
-        baz: 5
+          baz: 5
       )
       RUBY
     end
 
     context 'with a comment following the last key-value pair' do
       it 'corrects and leaves line breaks' do
-        src = <<-RUBY.strip_indent
```

Comment:
You make a good point that real code includes indentation. But for the scope of this PR, I would remove these alterations.

You could open this topic up as an issue and ask for input. Then should you get a green light from the code owner, I think a PR (or set of PRs) that make a more universal adjustment to this convention would be the best way to move forward.